src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -244,16 +218,75 @@ Status RedisStoreClient::AsyncMultiGet(
   RAY_CHECK(callback);
   if (keys.empty()) {
     callback({});
+    return Status::OK();
   }
   std::vector<std::string> true_keys;
   for (auto &key : keys) {
     true_keys.push_back(GenRedisKey(external_storage_namespace_, table_name, key));
   }
-  RAY_CHECK_OK(MGetValues(
-      redis_client_, external_storage_namespace_, table_name, true_keys, callback));
+  MGetValues(table_name, true_keys, callback);
   return Status::OK();
 }
 
+void RedisStoreClient::SendRedisCmd(std::vector<std::string> keys,
+                                    std::vector<std::string> args,
+                                    RedisCallback redis_callback) {
+  RAY_CHECK(!keys.empty());
+  std::string debug_redis_cmds;
+  auto counter = std::make_shared<std::atomic<size_t>>(0);
```

Comment:
counter -> num_ready_keys
we should also add some comments on what exactly num_ready_keys mean.